tests/pester-tests/features/CredentialProvider.Tests.ps1

```diff
@@ -0,0 +1,66 @@
+﻿Describe 'Ensuring credentials do not bleed from configured sources' -Tag CredentialProvider -ForEach @(

```

Comment:
Same comment here, as per the other PR. I think we need more words here as to what this test is doing, and why the asserts are set the way that they are.